certificate-policy.md

```diff
@@ -108,7 +108,7 @@ The primary goal of these Requirements is to enable efficient and secure electro
 
 
 ### 1.4.2 Prohibited Certificate Uses
-All Person identity certificates including but not limited to Person certificates used for digital signature, S/MIME, person authentication, and encryption. Trusted Role certificates MAY be issued from either the Root or Subordinate CA.
+All Person identity certificates including but not limited to Person certificates used for digital signature, S/MIME, person authentication, and encryption unless issued for specific infrastructure purposes as outlined in section 6.1.7.
```

Comment:
So section 6.1.7 says this:

> Root CA Private Keys SHALL NOT be used to sign Certificates except in the following cases:
> 
> 1. Self-signed Certificates to represent the Root CA itself;
> 2. Certificates for Subordinate CAs and Cross Certificates;
> 3. Certificates for infrastructure purposes (e.g. administrative role certificates, internal CA operational device certificates, and OCSP Response verification Certificates);

Is this edit saying we might issue person certificates from this root for use in validating Trusted Roles? That's not what I was expecting, and I think dilutes the intent of the hierarchy as an NPE root. The BRs don't seem to contain any references to Trusted Roles that expects them to be validated using PKI-based person certificates.